### DISCUSSION TITLE
Official app router example: TypeError: Cannot read properties of undefined (reading 'en')

### DISCUSSION THREAD

**Discussion by emil0519** [amannn/next-intl#1073] [Q&A]
### Description

I'm trying to learn next-intl with example code but stuck at this problem. Is there anything wrong with the official example?

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on GitHub and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

https://github.com/amannn/next-intl.git

### Reproduction description

Reproduction:
1. git clone https://github.com/amannn/next-intl.git
2. cd examples/example-app-router
3. npm i
4. npm run dev
5. Switch language with the select on top-right hand corner
6. Error throw:
![截圖 2024-05-17 下午2 27 28](https://github.com/amannn/next-intl/assets/97882056/ba5d3074-4557-47e3-8589-163ebe5e3b68)

### Expected behaviour

Switch language successfully without throwing error

**Comment by amannn** [comment]
Hmm, works for me without an issue. Maybe there's an issue with the monorepo setup, you can try to deploy/clone the example [as described in the README](https://github.com/amannn/next-intl/tree/main/examples/example-app-router#deploy-your-own).

I'll convert this to a discussion since it seems to be a usage question.
